packages/compass-web/src/entrypoint.tsx

```diff
@@ -95,6 +101,10 @@ type CompassWebProps = {
   >['onActiveWorkspaceTabChange'];
 
   initialPreferences?: Partial<AllPreferences>;
+
+  onLog?: LogFunction;
+  onDebug?: DebugFunction;
```

Comment:
Neither log, not debug will be used by mms initially, but the sandbox is using them, also it's easier to account for them right away when implementing logger and telemetry interface